src/dfx-core/src/config/model/dfinity.rs

```diff
@@ -407,19 +407,27 @@ impl CanisterTypeProperties {
     }
 }
 
-#[derive(Copy, Clone, Default, Debug, PartialEq, Eq, Serialize, Deserialize, JsonSchema)]
+#[derive(Clone, Default, Debug, PartialEq, Eq, Serialize, Deserialize, JsonSchema)]
 #[serde(rename_all = "lowercase")]
```

Comment:
```suggestion
#[serde(rename_all = "snake_case")]
```
Yes, the rest of the file uses `"lowercase"`, but these are all one word fields. We use `snake_case` where it makes a difference